docs/basic-features/image-optimization.md

```diff
@@ -191,6 +191,8 @@ You cannot use [styled-jsx](/docs/basic-features/built-in-css-support.md#css-in-
 
 You cannot use the `style` prop because the `<Image>` component does not pass it through to the underlying `<img>`.
 
+> An additional `raw` layout mode is provided which removes the wrapper element and allows the `style` prop. This mode still requires `height` and `width` and is recommended only for advanced use cases that aren't covered by the primary layout modes.
```

Comment:
Could we also add some examples for some reasonable use cases for "raw"? And how to avoid gotchas that could cause CLS?

```suggestion
> An additional `raw` layout mode is provided which removes the wrapper element and allows the `style` prop to be used. This mode still requires `height` and `width` and is recommended only for advanced use cases that aren't covered by the primary layout modes. When this mode is selected, it's possible to style the component in a way that causes CLS, so proceed with caution.
```